scripts/qa_crowbarsetup.sh

```diff
@@ -1414,47 +1456,73 @@ function enable_ssl_for_nova_dashboard()
     proposal_set_value nova_dashboard default "['attributes']['nova_dashboard']['ssl_no_verify']" true
 }
 
-function hacloud_configure_cluster_defaults()
+function hacloud_configure_cluster_members()
 {
-    clustertype=$1
+    local operation=$1
+    local clustername=$2
     shift
+    shift
+    # default is to add nodes
+    local op="+="
```

Comment:
The function we call is currently implemented in ruby, but this could change at any time. So I would not name it "ruby*". Just "operator" may be an option, but thats what "op" already stands for.
Is this really that important?